src/chainparams.cpp

```diff
@@ -213,16 +213,16 @@ class CTestNetParams : public CChainParams {
         consensus.vDeployments[Consensus::DEPLOYMENT_TAPROOT].nTimeout = 1628640000; // August 11th, 2021
         consensus.vDeployments[Consensus::DEPLOYMENT_TAPROOT].min_activation_height = 0; // No activation delay
 
-        consensus.nMinimumChainWork = uint256S("0x00000000000000000000000000000000000000000000064728c7be6fe4b2f961");
-        consensus.defaultAssumeValid = uint256S("0x00000000000163cfb1f97c4e4098a3692c8053ad9cab5ad9c86b338b5c00b8b7"); // 2143398
+        consensus.nMinimumChainWork = uint256S("0x00000000000000000000000000000000000000000000076f6e7cbd0beade5d20");
+        consensus.defaultAssumeValid = uint256S("0x0000000000000004877fa2d36316398528de4f347df2f8a96f76613a298ce060"); // 2344474
 
         pchMessageStart[0] = 0x0b;
         pchMessageStart[1] = 0x11;
         pchMessageStart[2] = 0x09;
         pchMessageStart[3] = 0x07;
         nDefaultPort = 18333;
         nPruneAfterHeight = 1000;
-        m_assumed_blockchain_size = 40;
+        m_assumed_blockchain_size = 42;
```

Comment:
On my node, it's only 32 GB. With 10% buffer, that's only 35.2, so this probably doesn't need to updated?

```
$ du -h --exclude=wallets --exclude=indexes testnet3/                                                                                                  04:23:29 PM
188K	testnet3/database
335M	testnet3/blocks/index
30G	testnet3/blocks
1.5G	testnet3/chainstate
32G	testnet3/
```

Reply:
I also get 29G for the blocks, plus 0.5 for blocks/index. That's on a freshly synced node. Perhaps a node that went through many reorgs uses a bit more.